### DISCUSSION TITLE
MediaWiki websites to embed Wikipedia pages

### DISCUSSION THREAD

**Discussion by joomleb** [mpratt/Embera#82] [Ideas]
Hi guys, 
Please, Can you consider to add support to a MediaWiki websites to embed Wikipedia pages.
It is a must for us - We found these resources that can be useful:
- [Stackoverflow](https://stackoverflow.com/questions/47057211/pulling-text-content-from-wikipedia-iframes) solutions by working with Wikipedia API
- [qlWiki](https://blackforest-software.com/index.php/en/nice-to-have/qlwiki/download) free joomla plugin support 
- [RPD Wiki](https://wordpress.org/plugins/rdp-wiki-embed/) Embed Wordpress plugin (as example)

**Comment by mpratt** [comment]
Hi @joomleb 
I answered on  #83
